config/initializers/session_store.rb

```diff
@@ -10,8 +10,10 @@
 servers = ApplicationConfig["REDIS_SESSIONS_URL"] || ApplicationConfig["REDIS_URL"]
 Rails.application.config.session_store :redis_store,
                                        key: ApplicationConfig["SESSION_KEY"],
+                                       domain: ApplicationConfig["APP_DOMAIN"],
                                        servers: servers,
                                        expire_after: expires_after,
                                        signed: true,
-                                       secure: ApplicationConfig["FORCE_SSL_IN_RAILS"] == "true",
+                                       secure: ENV["FORCE_SSL_IN_RAILS"] == "true",
```

Comment:
Why change from App config to ENV here?

Reply:
Oh I think I just changed that without really thinking about it because I was trying a bunch of different scenarios. I just changed it back to avoid an change that doesn't affect anything.